packages/contentful--create-contentful-app/lib/index.js

```diff
@@ -49,20 +48,20 @@ async function cloneTemplate(name, destination) {
   }
 }
 
-async function initProject() {
-  const appName = process.argv[3];
-
+async function initProject(appName, options) {
   try {
-    if (!appName) {
-      throw new Error(`Please provide a name for your app, e.g. \`${mainCommand} init my-app\``);
-    }
-
     if (!validateAppName(appName).validForNewPackages) {
       throw new Error(
         `Cannot create an app named "${appName}". Please choose a different name for your app.`
       );
     }
 
+    if (options.npm && options.yarn) {
+      throw new Error(
+        `Please provide either ${chalk.bold('--yarn')} or ${chalk.bold('--npm')} flag, but not both`
+      );
+    }
```

Comment:
When both flags are set, we want to use npm. That's the same as we do it when `--javascript `and `--typescript` is set.

Right? @fabe

Reply:
Yes, let's default npm in that case!

Reply:
Maybe we can log a warning like `Provided both --yarn and --npm, using `--npm.`?

Reply:
And also add the same warning to js vs ts conflict? WDYT?